equipment_handler/source/ina226.c

```diff
@@ -109,58 +75,15 @@ int INA226_RegisterGet(i2cBASE_t *i2c,
                          INA226_Register_TypeDef reg,
                          uint16_t *val)
 {
+
   uint8_t data[2];
 
+  if (i2c_Send(i2c, addr, 1, &reg) == -1) {return -1;}
```

Comment:
Maybe I'm a noob, but tbh I really am not a fan of having this as a single line!

Reply:
I'm going to leave this for now and make a new pr with clang-format configured, after we settle on a configuration in https://github.com/AlbertaSat/ex2_obc_software/pull/49